SmartDeviceLink/SDLTextAndGraphicManager.m

```diff
@@ -720,6 +336,7 @@ - (nullable SDLArtwork *)blankArtwork {
 - (void)sdl_displayCapabilityDidUpdate:(SDLSystemCapability *)systemCapability {
     // we won't use the object in the parameter but the convenience method of the system capability manager
     self.windowCapability = self.systemCapabilityManager.defaultMainWindowCapability;
+    [self sdl_updateTransactionQueueSuspended];
     
     // Auto-send an updated show
     if ([self sdl_hasData]) {
```

Comment:
I think that this may need to call `sdl_updateWithCompletionHandler` or else it won't be updated if it's not "dirty".